### PR TITLE
update SPERR package

### PR DESCRIPTION
**Pull request by shaomeng**
This PR updates the SPERR version.

### CODE REVIEW

**Review by robertu94 — APPROVED** [review]
Confirmed that it works on my machine and with LibPressio.

**Review by robertu94 — APPROVED** [review]

**Comment by robertu94** [comment]
I think something got goofed here

File: var/spack/repos/builtin/packages/sperr/package.py [hunk 1]
```diff
@@ -10,21 +10,25 @@ class Sperr(CMakePackage):
...
     depends_on("pkgconfig", type=("build"), when="+zstd")
 
     variant("shared", description="build shared libaries", default=True)
-    variant("zstd", description="use Zstd for more compression", default=True)
-    variant("openmp", description="use openmp for acceleration", default=True)
-
-    maintainers("shaomeng", "robertu94")
+    variant("zstd", description="use zstd for more compression", default=True)
+    variant("openmp", description="use openmp in 3D inputs", default=True)
+    variant("utilities", description="build SPERR CLI utilities", default=True)
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/sperr/package.py:31]: The default before this change was for this variant to essentially be `False` but now it is `True`.  Just want to confirm that is intentional.

**Review by tldahlgren — APPROVED** [review]
Confirmed the branch exists and the sha256 are correct.  Just a quick question about the default value of the new variant.

**Comment by tldahlgren** [comment]
I was able to successfully built the package at v0.6.2.

**Comment by tldahlgren** [comment]
@robertu94 Is this PR ready to merge as is?

File: var/spack/repos/builtin/packages/sperr/package.py (hunk 1 above)
Comment (shaomeng) [reply var/spack/repos/builtin/packages/sperr/package.py:31]: Yes, that's an intentional change. Also,  this PR is ready to  merge.

**Comment by robertu94** [comment]
Sorry for the delayed response, but yes it was ready 😄

**Pull request merged**